Lyubomyr add pet feature

KLCrow opened the pull request:
Реалізація сценарію add-pet.feature, та зміна вюверсів, для коректної його реалізації.

KLCrow committed:
Корегування вюх згідно сценарію New pet card feature

KLCrow committed:
Реалізація add-pet.feature

DimaSamodurov commented on spec/steps/pet_steps.rb:
```diff
@@ -42,6 +42,36 @@ module PetSteps
+  end
+
+  step 'я заповнив поля картки тварини' do
+    fill_in 'pet_name', :with => 'Tom'
```
Використовуй новий синтаксив для структур якщо явно не потрібно інакше.
fill_in 'pet_name', with: 'Tom'

DimaSamodurov commented on spec/steps/pet_steps.rb:
```diff
@@ -42,6 +42,36 @@ module PetSteps
+  end
+
+  step 'я маю побачити це оголошення' do
+    expect(page).to have_content 'Tom'
```
Краще вказати контейнер для тварини, наприклад, #pet-info. Якщо на кожній сторінці будуть присутні, скажімо, термінові оголошення, то пошук на всій сторінці може повернути не ту інформацію.

KLCrow committed:
Рефакторінт add-pet.feature

1. Корегування та рефакторінг кроків сценарію
2. Зміна views/pets/index - контейнер для списку тварин
